lib/active_admin/application.rb

```diff
@@ -105,6 +105,9 @@ def initialize
     # Set flash message keys that shouldn't show in ActiveAdmin
     inheritable_setting :flash_keys_to_except, ['timedout']
 
+    # Set the order and column to be used for ordering for comments
+    inheritable_setting :comment_order, "created_at ASC"
+
```

Comment:
All Comment's related settings are placed in the [comments.rb](https://github.com/activeadmin/activeadmin/blob/master/lib/active_admin/orm/active_record/comments.rb) file, simply place Line 109 in Line 10 of the comments.rb file.